corehq/apps/consumer_user/tests/test_views.py

```diff
@@ -230,12 +233,17 @@ def test_method_send_email(self):
                         case_properties={'email': 'email@changed.in'})
             self.assertEqual(ConsumerUserInvitation.objects.count(), 2)
             self.assertEqual(send_html_email_async.call_count, 2)
+            # Updating the case again with status other than sent or accepted should send email again
+            update_case(self.domain, case.case_id,
+                        case_properties={CONSUMER_INVITATION_STATUS: 'resend'})
+            self.assertEqual(ConsumerUserInvitation.objects.count(), 3)
```

Comment:
Could you check here that there's only one active invitation?

Reply:
Added the assertions